src/contribute/runner-guide.md

```diff
@@ -0,0 +1,1103 @@
+---
+layout: default
+title: "Runner Guide"
+permalink: /contribute/runner-guide/
+---
+
+# Runner Guide
```

Comment:
Done.

You have correctly surmised that I was trying to make it fit well. But actually when rendered it seems to fit fine with the longer, clearer, title.